src/Microsoft.Diagnostics.EventFlow.ServiceFabric/ServiceFabricDiagnosticPipelineFactory.cs

```diff
@@ -21,16 +21,32 @@ public static class ServiceFabricDiagnosticPipelineFactory
         public static readonly string FabricConfigurationValueReference = @"servicefabric:/(?<section>\w+)/(?<name>\w+)";
         public static readonly string FabricConfigurationFileReference = @"servicefabricfile:/(?<filename>.+)";
 
-        public static DiagnosticPipeline CreatePipeline(string healthEntityName, string configurationFileName = "eventFlowConfig.json")
+        public static DiagnosticPipeline CreatePipeline(
+            string healthEntityName, 
+            string configurationFileName = "eventFlowConfig.json",
```

Comment:
nit: I know this is not related to this change but can we make this a constant as well since we are touching this signature anyway. No big deal. Your call.